docs/UserGuide.md

```diff
@@ -148,8 +119,7 @@ Acceptable values for the keywords:
 * `ALL` (shows all student detail fields)
 
 Examples:
-* `list` displays the `SCHEDULE` list with all the lessons.
-* `list schedule` displays all the lessons with their `NAME`.
+* `list` and `list schedule` displays the `SCHEDULE` list with all the lessons with their `NAME`.
 * `list students` displays all the students with their `NAME` (including previously specified fields).
```

Comment:
+ "in alphabetical order"

Reply:
seems like elaine is at the bottom of the list instead of below david, are we fixing this?
<img width="423" alt="Screenshot 2023-11-04 at 8 26 07 PM" src="https://github.com/AY2324S1-CS2103T-T11-3/tp/assets/122242919/f6969339-e221-4cb6-add2-32a06d38c836">

Reply:
Made the necessary changes in the recent commit. Thank you!

Reply:
> seems like elaine is at the bottom of the list instead of below david, adding a new issue for this?
> <img alt="Screenshot 2023-11-04 at 8 26 07 PM" width="423" src="https://user-images.githubusercontent.com/122242919/280465812-f6969339-e221-4cb6-add2-32a06d38c836.png">

Thank you for pointing this out.Yes, will post an issue and work on it.